tests/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "env": {
+    "mocha": true
+  },
+  "rules": {
+    "no-with": 0,
+    "semi": 0
```

Comment:
Not sure if we want to keep `semi` enabled (or default value from inherited .eslintrc). How many missing semicolons do we have in the /tests/** directory?